clock.json

```diff
@@ -259,6 +351,78 @@
                "minute": 1441
             },
             "expected": true
+         },
+         {
+            "description": "clocks with minute overflow by several days",
+            "clock1": {
+               "hour": 2,
+               "minute": 2
+            },
+            "clock2": {
+               "hour": 2,
+               "minute": 4322
+            },
+            "expected": true
+         },
+         {
+            "description": "clocks with negative minute",
+            "clock1": {
+               "hour": 2,
+               "minute": 40
+            },
+            "clock2": {
+               "hour": 3,
+               "minute": -20
+            },
+            "expected": true
+         },
+         {
+            "description": "clocks with negative minute that wraps",
+            "clock1": {
+               "hour": 4,
+               "minute": 10
+            },
+            "clock2": {
+               "hour": 5,
+               "minute": -1490
+            },
+            "expected": true
+         },
+         {
+            "description": "clocks with negative minute that wraps multiple times",
+            "clock1": {
+               "hour": 6,
+               "minute": 15
+            },
+            "clock2": {
+               "hour": 6,
+               "minute": -4305
+            },
+            "expected": true
+         },
+         {
+            "description": "clocks with negative hours and minutes",
+            "clock1": {
+               "hour": 7,
+               "minute": 32
+            },
+            "clock2": {
+               "hour": -12,
+               "minute": -268
+            },
+            "expected": true
+         },
+         {
+            "description": "clocks with negative hours and minutes that wrap",
```

Comment:
Oops, both of those were supposed to be negative. It's harder to test these--I should have regenerated the Go tests using this; I'll go do that now.